### DISCUSSION TITLE
Unable to resolve dependance while setting up the project locally

### DISCUSSION THREAD

**Discussion by balaji119** [MoneyFox/MoneyFox.Windows#2974] [General]
I am facing the following error while building the project locally.

Severity	Code	Description	Project	File	Line	Suppression State
Error	NETSDK1141	Unable to resolve the .NET SDK version as specified in the global.json located at P:\MoneyFox\global.json.	MoneyFox.Ui	C:\Program Files\dotnet\sdk\7.0.302\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.DefaultItems.targets	127	

Can anyone help me resolve this?

**Comment by NPadrutt** [comment]
hey @balaji119 
what dotnet version do you have installed? You can check that via this command in the terminal: `dotnet --list-sdks`

We already are on the version that was released yesterday: 7.0.306
In case you don't have this already, there probably is a Visual Studio Update that installs this. Otherwise you can install it from the dot.net website or via winget: `winget install Microsoft.DotNet.SDK.7`

**Reply by balaji119** [reply]
Hey @NPadrutt 
Thanks for the solution. The version I had was 7.0.302. After installing the latest the build issue is resolved.

Now I am facing the following error while running. 

`DEP0700: Registration of the app failed. [0x80073CF0] error 0x80070003: Opening file from location: AppxManifest.xml failed with error: The system cannot find the path specified.`

I have added the certificate as mentioned in contributing.md
Can you help me resolve this?

**Reply by NPadrutt** [reply]
mmh, I quickly googled and found it reported here: https://learn.microsoft.com/en-us/answers/questions/1178661/how-to-fix-error-dep0700-maui-net7-0

It seems that is an error in MAUI on Windows. As a quick workaround for the moment: 
1. Go to the .csproj file and change line 61 from:
`        <MauiSplashScreen Include="Resources\Splash\splashscreen.svg" Color="#ea9972" BaseSize="128,128"/>`
to 
`        <MauiSplashScreen Include="Resources\Splash\splashscreen.svg" Color="#ea9972" BaseSize="208,208"/>`
2. Build and deploy the app
3. Revert the change

I have a PR running that pushes the work around mentioned in the discussion above. https://github.com/MoneyFox/MoneyFox/pull/2975
We'll see if that fixes it.

**Reply by balaji119** [reply]
I was using Debug Target as a`Windows Machine` and that was causing the issue. When I target an Android Emulator the application ran as expected.

**Reply by NPadrutt** [reply]
yeah, the issue seems to be only on windows. Although with the work around above it should also work there.
My guess is that MAUI for some reason doesn't detect that it has to generate the splash screen png and then a step later the build fails due since it can't find it.
